scripts/polkadot-launch/dali-dev-b.nix

```diff
@@ -0,0 +1,40 @@
+{ pkgs, polkadot-bin, composable-bin, composableParaId ? 2137 }:
+with pkgs;
+let builder = pkgs.callPackage ./network-builder.nix { };
+in {
+  result = builder.mk-shared-security-network {
+    relaychain = {
+      bin = "${polkadot-bin}/bin/polkadot";
+      chain = "rococo-local";
+      port = 40444;
+      wsPort = 19944;
+      count = 2;
+      flags = [
+        "--unsafe-ws-external"
+        "--unsafe-rpc-external"
+        "--rpc-external"
+        "--ws-external"
+        "--rpc-methods=Unsafe"
+      ];
+    };
+    parachains = [{
+      id = composableParaId;
+      port = 41200;
+      wsPort = 19988;
+      count = 3;
+      chain = "dali-dev";
```

Comment:
ParaId equals 2087 in `dali-dev` chain spec.
looks like, there will a problem with the registration of this parachain in relaychain.

Reply:
Dzmitry and I checked this. This isn't relevant.

Reply:
will clean up in next pr. saw block produced on different ports. seems like it influence only XCM, not blocks. because id is written in genesis.